hack/e2e.go

```diff
@@ -211,6 +211,20 @@ func run(deploy deployer) error {
 		if err := xmlWrap("Up", deploy.Up); err != nil {
 			return fmt.Errorf("starting e2e cluster: %s", err)
 		}
+		if *dump != "" {
```

Comment:
Actually, that may not be the best for large clusters, where `get nodes -oyaml` might be dicey in the first place. Hrm.